test/test_vmap.py

```diff
@@ -3527,6 +3523,49 @@ def test_random_inplace_not_batched(self, device, use_generator, randomness):
             op(passed, vmaped_value)
             assert torch.allclose(unvmaped_value, passed)
 
+    @parametrize('randomness', ['same', 'different', 'error'])
+    @parametrize('use_generator', [True, False])
+    @parametrize('batched_input', [True, False])
+    def test_distributions(self, device, randomness, use_generator, batched_input):
+        supported_ops = [
+            lambda _, t: torch.normal(t, 1., **kwargs),
+            lambda _, t: torch.normal(0., torch.abs(t), **kwargs),
+        ]
+
+        B0 = 4
+        seed = 1234567
+        generator = torch.Generator(device=device)
+        orig_state = generator.get_state()
+        generator = self.reset_random(generator, orig_state, use_generator, seed)
+
+        in_dims = (0, 0) if batched_input else (0, None)
+        args_shape = ((B0, B0), (B0, B0)) if batched_input else ((B0, B0), (B0,))
```

Comment:
Prefer to use B0, B1, etc to represent dimensions that are being vmapped over. We're probably not vmapping over both dimensions of each input right?